app/models/target.rb

```diff
@@ -1,9 +1,22 @@
 class Target < ApplicationRecord
   belongs_to :topic
-  belongs_to :user
+  belongs_to :user, counter_cache: true
+
+  MAX_ALLOWED_TARGETS = 10
```

Comment:
🙌 
Suggestion: We should potentially move this to the `User` model as they are the ones that cannot have more than 10 targets, and then on [this line](https://github.com/lestairon/target/pull/12/files#diff-dc424b3ecfa57a1e48fd9c66551ee25b407fcf61efd475b2248381228f958597R16) we could ask the user if they can have more targets, `errors.add(:user, 'exceeded number of targets limit') unless user&.can_add_more_targets?`